qa/L0_sequence_batcher/test.sh

```diff
@@ -332,6 +336,23 @@ for MODEL in $MODELS; do
       (cd queue_delay_models/$(basename $MODEL)_full && \
         sed -i "s/$(basename $MODEL)/$(basename $MODEL)_full/" config.pbtxt && \
         sed -i "s/minimum_slot_utilization: 0/minimum_slot_utilization: 1/" config.pbtxt)
+
+    # TODO: Enable single state buffer testing for sequence batcher
```

Comment:
still needed?

Reply:
Yes, I'm planning to add more testing for single state buffer later.